scipy/misc/pilutil.py

```diff
@@ -94,10 +94,10 @@ def bytescale(data, cmin=None, cmax=None, high=255, low=0):
         cscale = 1
 
     scale = float(high - low) / cscale
-    bytedata = (data * 1.0 - cmin) * scale + 0.4999
+    bytedata = (data * 1.0 - cmin) * scale + low
```

Comment:
I know it's not part of your changes, but it seems like the `* 1.0` is unnecessary here.